docs/docs/en/guide/installation/kubernetes.md

```diff
@@ -521,226 +521,227 @@ common:
-| `api.service.annotations`                                                         | `annotations` may need to be set when service.type is LoadBalancer                                                             | `{}`                                                  |
-|                                                                                   |                                                                                                                                |                                                       |
-| `ingress.enabled`                                                                 | Enable ingress                                                                                                                 | `false`                                               |
-| `ingress.host`                                                                    | Ingress host                                                                                                                   | `dolphinscheduler.org`                                |
-| `ingress.path`                                                                    | Ingress path                                                                                                                   | `/dolphinscheduler`                                   |
-| `ingress.tls.enabled`                                                             | Enable ingress tls                                                                                                             | `false`                                               |
-| `ingress.tls.secretName`                                                          | Ingress tls secret name                                                                                                        | `dolphinscheduler-tls`                                |
+| Parameter                                                            | Description                                                                                                                   | Default                               |
+|----------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------|---------------------------------------|
+| `timezone`                                                           | World time and date for cities in all time zones                                                                              | `Asia/Shanghai`                       |
+|                                                                      |                                                                                                                               |                                       |
```

Comment:
Is this format by spotless? If not, please remove the format change

Reply:
> Is this format by spotless? If not, please remove the format change

@ruanwenjun No, we haven't enabled spotless to fix md files.

Reply:
@ruanwenjun FYI https://github.com/apache/dolphinscheduler/pull/11384

Reply:
> Is this format by spotless? If not, please remove the format change

This is automatically modified by `IntelliJ Idea` to a uniform table width for better document readability.

Reply:
@rickchengx I've configured `Spotless` to format `markdown` automatically, see: https://github.com/apache/dolphinscheduler/pull/11384
For your convenience, you may rebase your code, make some changes in `kubernetes.md` and run `mvn spotless:apply`. In this way, `Spotless` will fix the format for you. I strongly suggest you configure the pre-commit hook following the instructions here: https://dolphinscheduler.apache.org/en-us/docs/dev/user_doc/contribute/development-environment-setup.html#code-style

Reply:
> @rickchengx I've configured `Spotless` to format `markdown` automatically, see: #11384 For your convenience, you may rebase your code, make some changes in `kubernetes.md` and run `mvn spotless:apply`. In this way, `Spotless` will fix the format for you. I strongly suggest you configure the pre-commit hook following the instructions here: https://dolphinscheduler.apache.org/en-us/docs/dev/user_doc/contribute/development-environment-setup.html#code-style

@EricGao888 , Thanks for the suggestion, I've rebased and used the `spotless` to apply the code style.